### ISSUE TITLE
Add publisher/journal: Wiley

### ISSUE THREAD

**Issue by mark02139** [CDL-Dryad/dryad-product-roadmap#2191]
**Name to appear on membership page:** 
John Wiley & Sons, Inc

**Payment plan:**
Wiley has recruited societies to participate in it's agreement with Dryad, thereby lowering their DPCs costs due to Wiley's aggregate volume.

One of the societies is new (Royal Entomological Society).
Three previously paid for their DPCs through direct agreements with Dryad, but will now have their DPCs sponsored by Wiley.
Wiley also has a number of proprietary journals.

For invoicing requirements, see https://github.com/CDL-Dryad/dryad-product-roadmap/issues/2190

**Journals included:** 
**Proprietary journals**
Diversity and Distribution
Ecology and Evolution
Environmental DNA
Evolutionary Applications
Global Ecology and Biogeography
Journal of Biogeography
Molecular Ecology
Molecular Ecology Resources
Global Change Biology
GCB Letters
Ecology Letters
Ethology

**Society Partners**
**British Ecological Society (moving to Wiley):**
Journal of Ecology
Journal of Animal Ecology
Journal of Applied Ecology
Methods in Ecology and Evolution
Functional Ecology
Ecological Solutions and Evidence
People and Nature

**Nordic Society Oikos (moving to Wiley):**
Oikos
Ecography
Journal of Avian Biology
Wildlife Biology
Nordic Journal of Botany

**Royal Entomological Society (new society):**
Ecological Entomology
Agricultural and Forest Entomology
Insect Conservation and Diversity
Insect Molecular Biology
Medical and Veterinary Entomology
Physiological Entomology
Systematic Entomology

**European Society for Evolutionary Biology (moving to Wiley)**
Journal of Evolutionary Biology

**Association for Tropical Biology and Conservation**
Biotropica

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/markkurtz_1/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/markkurtz_1/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Biotropica
--

</body>

</html>

**NB: I don't know who is responsible for assigning various email contacts or how that process is done.**

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**

**Email contacts to discuss integration of journal metadata with Dryad:**

Development tasks:
- [x] add member name to membership page
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [x] start integration process (track on new ticket)

@mariapraetzellis @ryscher @ahamelers @sfisher @jleighherzog

**Comment by ryscher** [comment]
@mark02139 
- GCB Bioenergy was attached to Wiley, but not listed above. I'm assuming we should keep it on the Wiley plan.
- GCB Letters does not seem to exist. I cannot find it listed anywhere on the web or in the ISSN registry.
- When we display sponsorship information in Dryad, should the society journals display as having the society sponsor, or Wiley?

**Comment by mark02139** [comment]
@ryscher , the list was actually sent to us by Pernille at Wiley, so not sure how to deal with discrepancies. 
I also can't find GCB Letters anywhere (though we have a Salesforce account for it).

Sponsorships: Wiley should be the sponsor, as that indicates who is responsible for DPC payment.
